tests/test_noninteractive_mode.py

```diff
@@ -0,0 +1,96 @@
+""" Non-interactive tests. """
+import subprocess
+from mssqltestutils import (
+    create_mssql_cli,
+    # create_test_db,
+    # clean_up_test_db,
+)
+
+# TODO: split up into multiple classes and create automated way to generate query files
+class TestNonInteractive:
+    """ Tests non-interactive features. """
+
+    file_root = './tests/test_query_results/'
```

Comment:
I don't think you can assume root to be the current directory. I think there a python API to get the current file path, from which you can infer the path.